README.md

```diff
@@ -8,7 +8,7 @@ Supported CloudEvents versions:
 Supported Protocols:
 - HTTP
 
-# **CloudEvents.Sdk** Module
```

Comment:
Minor to improve readability we could code highlight all occurrences of CloudEvents.Sdk,eg

```suggestion
# `**CloudEvents.Sdk**` Module
```